vars/runStackContinuousPipeline.groovy

```diff
@@ -54,6 +54,15 @@ def call() {
                     """
                 }
             }
+
+            stage('Sync images') {
+
+                steps {
+                    // We don't need to wait, and waiting takes up an executor slot
+                    build job: '/crossplane/sync-stacks', wait: false
```

Comment:
what exactly is this running? i'm not familiar with this syntax and i'm not sure exactly what is getting invoked here. can you point me to it?